base/features/data-hibernate-jpa/skeleton/src/main/resources/application.yml

```diff
@@ -1,11 +1,3 @@
-datasources:
-  default:
-    url: jdbc:h2:mem:devDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE
-    driverClassName: org.h2.Driver
-    username: sa
-    password: ''
-    schema-generate: CREATE_DROP
```

Comment:
I think you misunderstood sorry. Only `schema-generate` should have been deleted

Reply:
I did misunderstand, I have put everything back other than the schema-generate